wiki/Welcome/en.md

```diff
@@ -37,3 +14,18 @@ To get started, see the [osu!wiki Contribution Guide](/wiki/owcg).
   - [osu!catch](/wiki/osu!catch/#getting-started)
   - [osu!mania](/wiki/osu!mania/#getting-started)
 - [Multi](/wiki/Multi/#getting-started)
+- [Beatmapping](/wiki/Beatmapping/#getting-started)
+- [Modding](/wiki/Modding/#getting-started)
+
+## Current events
+
+<!--There are no active events within the osu! community at this time.-->
+- [Aspire Stage Three](https://osu.ppy.sh/home/news/2017-09-21-aspire-stage-three-osucatch-begins)
+- [osu!idol 2017 (Auditions)](https://osu.ppy.sh/home/news/2017-08-14-osu-idol-2017-auditions-now-open)
+- [Yuzu's New Look Contest](https://osu.ppy.sh/home/news/2017-08-21-yuzus-new-look-contest)
+
+## osu!wiki - How you can help!
+
+Since 26. August 2016, the osu!wiki has moved to a GitHub repository. After this change, the workflow has drastically changed. All contributions now go through the [osu-wiki GitHub repository](https://github.com/ppy/osu-wiki) and are reviewed before being approved and merged to the osu!wiki.
```

Comment:
L29, Possible formatting error, consider using ``DD, MonthName, YYYY``

Reply:
from the ASC, the current is the correct format